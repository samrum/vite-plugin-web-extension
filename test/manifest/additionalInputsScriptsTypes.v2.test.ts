import { getResourceDir, runManifestV2Test } from "./manifestTestUtils";

const resourceDir = getResourceDir("additionalInputsScriptsTypes");

runManifestV2Test("additionalInputsScriptsTypes", () => ({}), {
  additionalInputs: {
    scripts: [`${resourceDir}/script1.js`, `${resourceDir}/script2.ts`],
  },
});
