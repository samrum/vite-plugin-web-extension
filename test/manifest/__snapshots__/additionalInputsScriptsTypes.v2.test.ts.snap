// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`additionalInputsScriptsTypes - Manifest V2 1`] = `
[
  {
    "fileName": "manifest.json",
    "name": undefined,
    "source": "{
  "version": "1.0.0",
  "name": "Manifest Name",
  "manifest_version": 2,
  "web_accessible_resources": [
    "test/manifest/resources/additionalInputsScriptsTypes/script1.js",
    "test/manifest/resources/additionalInputsScriptsTypes/script2.js"
  ]
}",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsTypes/script1.js",
    "name": undefined,
    "source": "console.log("script1");
",
    "type": "asset",
  },
  {
    "fileName": "test/manifest/resources/additionalInputsScriptsTypes/script2.js",
    "name": undefined,
    "source": "console.log("script2");
",
    "type": "asset",
  },
]
`;
